edit_course/tests.py

```diff
@@ -33,8 +33,9 @@ def test_course_clone(self):
             'key_month': 'Test',
         }, follow=True)
         self.assertEqual(response.status_code, 200)
-        self.assertNotContains(response, "alert alert-danger")
-        self.assertContains(response, "alert alert-success")
+        stripped_content = response.content.decode("utf-8").replace("modal-submit-error alert alert-danger", "")
+        self.assertNotIn("alert alert-danger", stripped_content)
+        self.assertIn("alert alert-success", stripped_content)
```

Comment:
``base.html`` now includes a hidden ``div`` with ``class="modal-submit-error alert alert-danger"``.
It has to be stripped away before asserting in the course cloning tests so that the tests don't fail, because they think course cloning failed and rendered a different alert.